conda/base/context.py

```diff
@@ -327,6 +327,7 @@ class Context(Configuration):
     number_channel_notices = ParameterLoader(PrimitiveParameter(5, element_type=int))
     _verbosity = ParameterLoader(
         PrimitiveParameter(0, element_type=int), aliases=('verbose', 'verbosity'))
+    experimental_jlap = ParameterLoader(PrimitiveParameter(False))
```

Comment:
One Thing I Regret from our libmamba experiment is having used `experimental` in the flags and object names. It caused unnecessary disruption and tons of deprecation warnings when we removed it. We were given the following advice but had to ignore it because we just wanted to release it without further changes. That was a mistake.

The advice was to have a `--experimental` namespace, that takes features that should be turned on. Only a single flag, only a single container in the `context`. This is what `pip` does with `--experimental-feature`.

Reply:
It looks like I can do an append_list argument... and collect a list / set of feature strings

Reply:
Or possibly a custom Action() class.

Reply:
For reference, `pip`'s implementation: https://github.com/pypa/pip/blob/9aa422da16e11b8e56d3597f34551f983ba9fbfd/src/pip/_internal/cli/cmdoptions.py#L976

Reply:
Thanks! I did find the choices argument and made this change. https://github.com/conda/conda/pull/12090/commits/1c7be2c192db4ad879d16c8e2b46cd4b932dce24

It is smart to have pip's naming that doesn't need to change if it becomes non-experimental though.